### ISSUE TITLE
[1.18.2] Sometimes crashes on startup

### ISSUE THREAD

**Issue by java-jambalaya** [Majrusz/MajruszsProgressiveDifficultyMod#105]
- **Game Version: 1.18.2**
- **Forge Version: 40.1.68**
- **Majrusz's Progressive Difficulty Version: 1.4.3**
- **Majrusz's Library Version: 2.7.2**

I can't recreate this crash consistently, but sometimes loading the game will fail and yield the following message:

```
Majrusz's Progressive Difficulty (majruszsdifficulty) encountered an error during the sided_setup event phase
java.util.ConcurrentModificationException: null
```

Have only been able to recreate this with other mods running, but as I can't recreate this consistently I don't know if a specific mod incompatibility could be the cause. If that is the case, the minimum I was able to recreate this crash with was the following list:
```
majruszs-difficulty-1.18.2-1.4.3.jar
TerraBlender-forge-1.18.2-1.1.0.102.jar
forge-1.18.2-40.1.68-universal.jar
curios-forge-1.18.2-5.0.7.1.jar 
majrusz-library-1.18.2-2.7.2.jar
jei-1.18.2-9.7.1.232.jar
```

**Comment by Majrusz** [comment]
This issue has been fixed for 1.19.1 but it was not backported yet. Once I backport the latest 1.19.1 release (probably by the end of next week), this should no longer be an issue. Anyway thanks for the report 😄

**Comment by Majrusz** [comment]
Fixed in the latest 1.4.4 release
